src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -790,24 +789,19 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
 
             **Examples:**
 
-            `appName RLIKE 'z.*|q.*''`
+            `appName RLIKE r'z.*|q.*''`
+            
+            `hostname RLIKE r'ip-10-351-[0-2]?[0-9]-.*'`
 
             * `z-app`
             * `q-app`
-
-            `hostname RLIKE 'ip-10-351-[0-2]?[0-9]-.*'`
```

Comment:
I think we still want this line here,  not above. It relates to the bullets below it. We just don't want it as one of the bullet points.

Reply:
Fixed!